data/format.go

```diff
@@ -159,6 +159,7 @@ var encodings = map[enc]string{
 	enc{ST_HASH256, 20}: "TicketID",
 	enc{ST_HASH256, 21}: "Digest",
 	enc{ST_HASH256, 22}: "Channel",
+	enc{ST_HASH256, 23}: "CheckID", // What is the "23" here???
```

Comment:
https://github.com/ripple/rippled/blame/eaff9a0e6aec0ad077f118501791c7684debcfd5/src/ripple/protocol/impl/SField.cpp#L174

Looks like you were out by one :-) Probably worth adding `sfConsensusHash` in a separate PR.